packages/ucast-prisma/tests/masks.test.ts

```diff
@@ -0,0 +1,157 @@
+import { describe, it, expect } from "vitest";
+import { project, mask, MaskRule } from "../src";
+
+describe("masking", () => {
+  it("works for complete example", () => {
+    const tickets = [
+      { id: 0, descr: "something wrong", reporter: "John Doe", assignee: null },
+      {
+        id: 1,
+        descr: "something else wrong",
+        reporter: "Jane Doe",
+        assignee: 12,
+      },
+      {
+        id: 2,
+        descr: "everything wrong",
+        reporter: "Joanna Doe",
+        assignee: 100,
+      },
+    ];
+    const masks = {
```

Comment:
- [ ] add a test case for `{ "tickets.reporter": {} }` which is unavoidable when using rego for this 😄